bin/check_tron_jobs.py

```diff
@@ -122,7 +137,7 @@ def compute_check_result_for_job(client, job):
     kwargs = {
         "name": "check_tron_job.{}".format(job['name']),
         "team": 'noop',
-        "notification_email": "kwa+checktron@yelp.com",
+        "notification_email": job['monitoring'].get('notification_email', 'unspecified'),
```

Comment:
whoops!

Somewhere, we need to be merging in the `monitoring` second in here directly and updating the kwargs dictionary. This way we get all the options (like team, runbook, alert_after, etc) instead of hand picking keys.

We can't do it this way, because we'll be sending a lot of emails to `unspecified@yelp.com`!

Reply:
I think it will only send email to unspecified :p
But yeah, hand picking keys is not a good idea.